API.md

```diff
@@ -457,16 +456,16 @@ Returns:
 
 Parameters:
 
-- `choirId` - the id of the choir whose song is being rendered (required)
-- `songId` - the id of the song being rendered (required)
+- `partId` - the id of song part that triggered the render (required)
 
 Returns:
 
 ```js
 {
    ok: true,
    render: {
-     ... render object ...
+      status: 'new',
+      date: '2020-08-29T10:20:59.000Z'
```

Comment:
Would it be possible to use Unix time? It'll make it easier to sort results chronologically if ever needed and would be in line with other records already in the database

Reply:
This is the same form date used elsewhere in the API - and it sorts into date/time order too.

```sh
grep 'toISOString' *.js               
postChoir.js:      createdOn: now.toISOString(),
postChoir.js:        joined: now.toISOString(),
postChoirJoin.js:    joined: now.toISOString(),
postChoirSong.js:      createdOn: now.toISOString()
postChoirSongPart.js:      createdOn: now.toISOString(),
postRender.js:      date: new Date().toISOString()
postUser.js:      createdOn: now.toISOString()
```

Reply:
Ah, you're right. For some reason I thought we were using unix time for the consumers. Sorry!